src/isort/sorting.rs

```diff
@@ -37,10 +37,14 @@ pub fn cmp_modules(alias1: &AliasData, alias2: &AliasData) -> Ordering {
 }
 
 /// Compare two member imports within `StmtKind::ImportFrom` blocks.
-pub fn cmp_members(alias1: &AliasData, alias2: &AliasData) -> Ordering {
-    prefix(alias1.name)
-        .cmp(&prefix(alias2.name))
-        .then_with(|| cmp_modules(alias1, alias2))
+pub fn cmp_members(alias1: &AliasData, alias2: &AliasData, order_by_type: bool) -> Ordering {
+    if order_by_type {
+        prefix(alias1.name)
+            .cmp(&prefix(alias2.name))
+            .then_with(|| cmp_modules(alias1, alias2))
+    } else {
+        cmp_modules(alias1, alias2)
+    }
```

Comment:
Maybe a bit weird that `order_by_type` essentially turns this into a different existing function.
Is it better to select which function is used somewhere in `mod.rs`?

Reply:
I think this is ok. The current setup is a bit confusing as-is anyway (mostly just the nomenclature isn't that clear), but I like that this at least encapsulates the member sorting in one place.